### ISSUE TITLE
My conda_list() is empty   

### ISSUE THREAD

**Issue by AnthoOzier** [rstudio/reticulate#718]
I upgraded R from R3.4 to R3.6 and I'm not able to import my conda env anymore (I'm using miniconda ). My `conda_list()`  is empty  :

``` 
[1] name   python
<0 rows> (ou 'row.names' de longueur nulle)
```
What I do not understand is that I can specify the python of the conda env I want to use with `use_python(' path' )` and it works, but I would prefer using use_condaenv('env'). 
Any advice to fix this ?

**Comment by skeydan** [comment]
Do you have any environments created? E.g. could you do a `conda list` in the terminal?

If not, can you try creating one using `conda_list`?

**Comment by AnthoOzier** [comment]
I assume you asked for a `conda env list`, anyway, both `conda env list` and `conda list` give results on terminal. I have a lot of packages and five conda environments, so I think the problem is something wrong with my R settings.

**Comment by skeydan** [comment]
yes I meant `conda env list` ... So what are your R-related settings (`.Rprofile`, `.Renviron`, or anything on system level)?

Does the above happen from `R` and/or `RStudio`? What's the `PATH` as seen from R? (Sys.getenv("PATH"))?

What's your OS and where's `conda` installed? 

Oh, and something else, what happens if you use `conda activate someenv` before calling `R` / `RStudio`?

**Comment by AnthoOzier** [comment]
Thank you for your help ! 
I am on Ubuntu 18.04, I have no .Rprofile file and my .Renviron is empty except for the proxy I use.
My problem is with Rstudio and here is my `Sys.getenv("PATH")`: 
```
"/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin"
```
And conda is here :  
```
/home/anthony/miniconda3/bin/conda
```
Does it have to be in my system env "PATH"? 

And for your last question, activating my conda env before launching Rstudio gives me an Rstudio where not any of my R packages is installed (not surprising I guess).

**Comment by skeydan** [comment]
> activating my conda env before launching Rstudio gives me an Rstudio where not any of my R packages is installed (not surprising I guess).

totally surprising, to me! :-)
Are you using `renv` to manage your R environments? Or, totally different thought, how did you install `RStudio` - from `conda` or standalone?

> Does it have to be in my system env "PATH"?

I just installed a fresh Miniconda and at the end I got asked whether a wanted to run `conda init`, which adds that whole stuff to `bashrc` ... but it also told me I could run 

```
conda config --set auto_activate_base false
```

to avoid activating the default env at startup, which I did. After that, one still needs to run `conda activate` manually, but yes, `condabin` is on the PATH now.

 (see also https://stackoverflow.com/questions/54429210/how-do-i-prevent-conda-from-activating-the-base-environment-by-default)
